fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePartitioner.java

```diff
@@ -501,40 +529,62 @@ public static long getTo(@Nonnull LucenePartitionInfoProto.LucenePartitionInfo p
     /**
      * Re-balance full partitions, if applicable.
      *
-     * @return void future
+     * @param start The continuation at which to resume rebalancing, as returned from a previous call to
+     * {@code rebalancePartitions}.
+     * @return a tuple of the next group that needs repartitioning
```

Comment:
I think this could be worded more like `start` is. That the continuation state is a tuple advancing by groups isn't, that I can see, part of the contract of this function. Which is roughly, keep calling this until you get an ended continuation. It could do part of a group or multiple groups or whatever it wants inside without changing that.